std/traits.d

```diff
@@ -6130,8 +6130,27 @@ deprecated
 }
 
 /**
- * Detect whether $(D T) is a struct, static array, or enum that is implicitly
- * convertible to a string.
+  $(RED Warning: This trait will be deprecated as soon as it is no longer used
+                 in Phobos. For a function parameter to safely accept a type
+                 that implicitly converts to string as a string, the conversion
+                 needs to happen at the callsite; otherwise, the conversion is
+                 done inside the function, and in many (most?) cases, that
```

Comment:
I would do `s/many (most?)/most/` - the official documentation shouldn't speculate.